packages/roosterjs-editor-dom/lib/table/VTable.ts

```diff
@@ -120,8 +122,9 @@ export default class VTable {
                 });
             });
             if (this.formatInfo) {

```

Comment:
I think if you modify this if to
`if(this.formatInfo && !isResized)`
we wont apply format when resizing. And no need to modify applyFormat nor add shouldApplyFormatToCell

Reply:
The cell shade is also been replace when we edit the table and after editing the table we need apply the styles and 
 do not replace the cell shade.
![cellShade2](https://user-images.githubusercontent.com/87443959/152570251-6087954d-78d3-42a1-9358-760b834f81c1.gif)